.travis.yml

```diff
@@ -1,20 +1,19 @@
-  dist: trusty
+dist: trusty
 sudo: required
 language: python
 services:
   - mongodb
 python:
   - "3.6"
 env:
-  - POCS=$TRAVIS_BUILD_DIR PANDIR=/var/panoptes PANUSER=$USER ARDUINO_VERSION=1.8.1
-before_cache:
-    - sudo mkdir -p $PANDIR/astrometry/data && sudo chmod -R 777 $PANDIR
-cache:
-  pip: true
-  directories:
-    - $PANDIR/astrometry/data
+  - POCS=$TRAVIS_BUILD_DIR 
+  - PANDIR=/var/panoptes 
+  - PANUSER=$USER 
+  - ARDUINO_VERSION=1.8.1
 before_install:
-    - sudo mkdir -p $PANDIR/astrometry/data && sudo chmod -R 777 $PANDIR
+    - sudo mkdir -p $PANDIR && sudo chmod -R 775 $PANDIR
+    - sudo chown -R $PANUSER:$PANUSER $PANDIR
+    - sudo addgroup $USER $PANUSER
```

Comment:
Doesn't addgroup require logging out and back in before it will take effect?

Reply:
Removed as per above.